### ISSUE TITLE
RSpec let for assigning multiline hash leads to BlockDelimiter complaining

### ISSUE THREAD

**Issue by nakulpathak3** [bbatsov/rubocop#3745]
```ruby
let(:random_hash) {
  {
    key1: value1,
    key2: value2,
    key3: value3,
    key4: value4,
  }
}
```
In a block like this, Rubocop's BlockDelimiter cop complains with error - `Avoid using {...} for multi-line blocks.`

## Expected behavior

It should not have raised the error at all.

## Actual behavior

It raised the BlockDelimiter cop error `Avoid using {...} for multi-line blocks.`.

## Steps to reproduce the problem

Copy paste the above `let(:random_hash)` code I posted above to any rspec file and run `rubocop`.

## RuboCop version

```
$ rubocop -V
0.45.0 (using Parser 2.3.3.0, running on ruby 2.3.1 x86_64-darwin15)
```

**Comment by nakulpathak3** [comment]
Nevermind..

**Comment by BALOCH1** [comment]
@nakulpathak3 you can use do end block to avoid this. 

```
let(:random_hash) do
  {
    key1: value1,
    key2: value2,
    key3: value3,
    key4: value4,
  }
end
```
